packages/amplication-notifications-service/src/providers/secrets/base/secretsManager.service.base.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { ConfigService } from '@nestjs/config';
+import { mock } from 'jest-mock-extended';
+import { SecretsManagerServiceBase } from './secretsManager.service.base';
+
+describe('Testing the secrets manager base class', () => {
+  const SECRET_KEY = 'SECRET_KEY';
+  const SECRET_VALUE = 'SECRET_VALUE';
+  const configService = mock<ConfigService>();
+  const secretsManagerServiceBase = new SecretsManagerServiceBase(
+    configService
+  );
+  beforeEach(() => {
+    configService.get.mockClear();
+  });
+  it('should return value from env', async () => {
+    //ARRANGE
+    configService.get.mockReturnValue(SECRET_VALUE);
+    //ACT
+    const result = await secretsManagerServiceBase.getSecret(SECRET_KEY);
+    //ASSERT
+    expect(result).toBe(SECRET_VALUE);
+  });
+  it('should return null for unknown keys', async () => {
+    //ARRANGE
+    configService.get.mockReturnValue(undefined);
+    //ACT
+    const result = await secretsManagerServiceBase.getSecret(SECRET_KEY);
+    //ASSERT
+    expect(result).toBeNull();
+  });
+  it('should throw error if dont get key', () => {
```

Comment:
There's a typo here, it should be:
`should throw an error if key could not be found`

Reply:
this is an auto-generated file by Amplicaton, but I will fix it (we should fix it also in the generated app)